components/Nav/SearchWithAlgolia.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'
+import Router from 'next/router'
+
+import Search from './Search'
+
+let docsearch
+if (process.browser) {
+  docsearch = require('docsearch.js')
+}
+
+class SearchWithAlgolia extends Component {
+  isDocs = true
+
+  componentWillMount() {
+    this.isDocs = process.browser && Router.pathname.startsWith('/docs')
```

Comment:
I don't see why this should have `process.browser`? Isn't the `router` universal?

Reply:
No, it's not.

Reply:
Alright, then that's fine :+1:

Reply:
but it supposed to be.